### DISCUSSION TITLE
Output error logs

### DISCUSSION THREAD

**Discussion by Abhi-SimuTech** [ansys/pyaedt#3743] [Q&A] [answered]
Hi,

Is there an example showing how to output error log to an external file for later investigation! 

I have a script that creates a large geometry in HFSS and sometimes crashes (not always). I wish to output the error logs to troubleshoot for this issue.

Your suggestions will be appreciated.

Regards
Abhi

**Comment by SMoraisAnsys** [comment, marked as answer]
Hi, the first approach that comes to mind is to ensure that you have enabled the log file.
For example :
```python
from pyaedt.generic.general_methods import settings
settings.enable_global_log_file = True
```
Then a file should be generated in the repertory you selected during the installation of AEDT.
In my case that is "C:\Users\$USERNAME\AppData\Local\Temp\pyaedt_$USERNAME.log" where you have to fill your own username :)
Hope that helps you, let me know if that's not enough.
